lib/hostel/site.rb

```diff
@@ -15,7 +15,9 @@ def domain
     end
     
     # Constructs a url for a path on this site
-    def build_path(path, secure: true, path_args: {})
+    # Note:  request_host is ignored now, but it is here for backward
+    # compatibility with old callers.
```

Comment:
I disagree. Leaving those in there seems batshit bonkers. You can find ALL the places by running `ack "build_path"` across each of the (5) repos that use this gem. There are not that many. I'm confident in your ability to update all of them, and optimistic that you will also remove the description of this feature from the README since it is no longer supported